src/Resizer/ImagineCompatibleResizerTrait.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Sonata\MediaBundle\Resizer;
+
+use Imagine\Image\ImageInterface;
+use Imagine\Image\ImagineInterface;
+
+/**
+ * This trait used to provide compatibility with Imagine >= 1.0.0.
```

Comment:
Why use the past tense here? Did you mean to write this:
```suggestion
 * This trait is used to provide compatibility with Imagine >= 1.0.0.
```